File: September/12th September,2024/Leetcode(1684).py
class Solution:
    def countConsistentStrings(self, allowed: str, words: List[str]) -> int:
        # p=Counter(allowed)
        ans=0
        p=set(allowed)
        for i in words:
            flag=True
            for j in i:
                if j not in p:
                    flag=False
                    break
            if flag:
                ans+=1
        return ans

File: September/12th September,2024/Leetcode(1684).cpp
class Solution {
public:
    int countConsistentStrings(string allowed, vector<string>& words) {
        unordered_map<char,int>mp;
        for(char s:allowed){
            mp[s]++;
        }
        int ans=0;
        for(string i:words){
            bool flag=true;
            for(char ch:i){
                if(mp[ch]==0) flag=false;
            }
            if(flag) ans+=1;
        }
        return ans;
    }
};